modules/gapi/test/common/gapi_core_tests_inl.hpp

```diff
@@ -1297,7 +1297,7 @@ TEST_P(PhaseTest, AccuracyTest)
     // Comparison //////////////////////////////////////////////////////////////
     // FIXME: use a comparison functor instead (after enabling OpenCL)
     {
-        EXPECT_EQ(0, cvtest::norm(out_mat_ocv, out_mat_gapi, NORM_INF));
+        EXPECT_LT(cvtest::norm(out_mat_ocv, out_mat_gapi, NORM_INF), 4e-6);
```

Comment:
Please put this ARM check under:
```
#if defined(__aarch64__) || defined(__arm__)
```

guard and keep original check "as is".


BTW, almost all failures on ARM64 are about the "Fluid" backend.
CPU (OpenCV) backend results should stay the same.
It make sense to G-API team implement the "FIXME" comment about the comparison functor as test parameter later.